plugins/modules/ec2_instance.py

```diff
@@ -244,6 +244,14 @@
       - Whether to enable termination protection.
       - This module will not terminate an instance with termination protection active, it must be turned off first.
     type: bool
+  hibernation_options:
+    description:
+      - Indicates whether an instance is enabled for hibernation.
+        Refer U(https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/hibernating-prerequisites.html)
+        for Hibernation prerequisits.
```

Comment:
If this option requires a specific botocore version you should add a note here.

Reply:
This option is available in the latest and the versions we support.